src/watertap_contrib/reflo/property_models/air_water_equilibrium_properties.py

```diff
@@ -0,0 +1,1843 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+"""
+Air-water equilibrium property package
+"""
+from copy import deepcopy
+
+# Import Python libraries
+import idaes.logger as idaeslog
+
+from enum import Enum, auto
+import itertools
+
+# Import Pyomo libraries
+from pyomo.environ import (
+    Constraint,
+    Expression,
+    Reals,
+    Set,
+    Suffix,
+    NonNegativeReals,
+    Var,
+    Param,
+    exp,
+    log10,
+    value,
+    check_optimal_termination,
+)
+from pyomo.environ import units as pyunits
+from pyomo.common.config import ConfigValue, In
+from pyomo.util.calc_var_value import calculate_variable_from_constraint
+from watertap.core.util.model_diagnostics.infeasible import *
+
+# Import IDAES cores
+from idaes.core import (
+    declare_process_block_class,
+    MaterialFlowBasis,
+    PhysicalParameterBlock,
+    StateBlockData,
+    StateBlock,
+    MaterialBalanceType,
+    EnergyBalanceType,
+)
+from idaes.core.base.components import Solute, Solvent, Component
+from idaes.core.base.phases import (
+    LiquidPhase,
+    VaporPhase,
+    PhaseType as PT,
+)
+from idaes.core.util.constants import Constants
+from idaes.core.util.initialization import (
+    fix_state_vars,
+    revert_state_vars,
+    solve_indexed_blocks,
+)
+from idaes.core.util.misc import extract_data
+from idaes.core.solvers import get_solver
+from idaes.core.util.misc import add_object_reference
+from idaes.core.util.model_statistics import (
+    degrees_of_freedom,
+    number_unfixed_variables,
+)
+from idaes.core.util.exceptions import (
+    ConfigurationError,
+    InitializationError,
+)
+import idaes.core.util.scaling as iscale
+
+from watertap.core.util.scaling import transform_property_constraints
+
+boltzmann = pyunits.convert(
+    Constants.boltzmann_constant,
+    to_units=(pyunits.g * pyunits.cm**2) / (pyunits.second**2 * pyunits.degK),
+)
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+__author__ = "Kurban Sitterley"
+
+
+class MolarVolumeCalculation(Enum):
+    none = auto()
+    TynCalus = auto()
+
+
+class LiqDiffusivityCalculation(Enum):
+    none = auto()
+    HaydukLaudie = auto()
+
+
+class VapDiffusivityCalculation(Enum):
+    none = auto()
+    WilkeLee = auto()
```

Comment:
Would benefit from some minimal documentation (doc strings)

Reply:
Done. Full documentation will be provided in readthedocs.